<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RRcmp</class>
 <widget class="QMainWindow" name="RRcmp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RRcmp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>RRcomparator v3.0</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Specimen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_spec">
        <item>
         <property name="text">
          <string>inner#60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>inner#61</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>inner#62</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>outer#93</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>outer#97</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>outer#99</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Parameter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_par">
        <item>
         <property name="text">
          <string>Height (z)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SlopeX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SlopeY</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>devZ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>devSlopeX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>devSlopeY</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>Load file and </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_SWrealignment">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>keep original</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SW realignment by devZ_ideal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SW realignment by devZ_exp</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>common range: Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_min"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string> Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_max"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab Parameter</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>P3</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QLineEdit" name="lineEdit_DP4"/>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_0">
            <property name="text">
             <string>ENEA</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLineEdit" name="lineEdit_Fmin"/>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEdit_FP3"/>
          </item>
          <item row="5" column="5">
           <widget class="QLineEdit" name="lineEdit_Nmin"/>
          </item>
          <item row="3" column="6">
           <widget class="QLineEdit" name="lineEdit_Fmax"/>
          </item>
          <item row="5" column="6">
           <widget class="QLineEdit" name="lineEdit_Nmax"/>
          </item>
          <item row="5" column="7">
           <widget class="QPushButton" name="pushButton_map_3">
            <property name="text">
             <string>plot</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="lineEdit_IP4"/>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Max</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Min</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="lineEdit_SP2"/>
          </item>
          <item row="6" column="4">
           <widget class="QLineEdit" name="lineEdit_SP4"/>
          </item>
          <item row="4" column="6">
           <widget class="QLineEdit" name="lineEdit_Dmax"/>
          </item>
          <item row="5" column="4">
           <widget class="QLineEdit" name="lineEdit_NP4"/>
          </item>
          <item row="2" column="5">
           <widget class="QLineEdit" name="lineEdit_Emin"/>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEdit_EP3"/>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>NREL</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>DLR</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="lineEdit_EP4"/>
          </item>
          <item row="6" column="3">
           <widget class="QLineEdit" name="lineEdit_SP3"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_EP1"/>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_SP1"/>
          </item>
          <item row="5" column="3">
           <widget class="QLineEdit" name="lineEdit_NP3"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>P2</string>
            </property>
           </widget>
          </item>
          <item row="4" column="7">
           <widget class="QPushButton" name="pushButton_map_2">
            <property name="text">
             <string>plot</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_IP1"/>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>SANDIA</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>P1</string>
            </property>
           </widget>
          </item>
          <item row="6" column="5">
           <widget class="QLineEdit" name="lineEdit_Smin"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_DP1"/>
          </item>
          <item row="3" column="7">
           <widget class="QPushButton" name="pushButton_map_1">
            <property name="text">
             <string>plot</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_IP3"/>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="lineEdit_NP2"/>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="lineEdit_DP3"/>
          </item>
          <item row="2" column="7">
           <widget class="QPushButton" name="pushButton_map_0">
            <property name="text">
             <string>plot</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="lineEdit_DP2"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_FP1"/>
          </item>
          <item row="2" column="6">
           <widget class="QLineEdit" name="lineEdit_Emax"/>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_1">
            <property name="text">
             <string>FISE</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="lineEdit_FP4"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_IP2"/>
          </item>
          <item row="6" column="6">
           <widget class="QLineEdit" name="lineEdit_Smax"/>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="lineEdit_FP2"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>P4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_EP2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Ideal</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_NP1"/>
          </item>
          <item row="6" column="7">
           <widget class="QPushButton" name="pushButton_map_4">
            <property name="text">
             <string>plot</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QLineEdit" name="lineEdit_Dmin"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab XY ranges</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="5" column="4">
           <widget class="QLineEdit" name="lineEdit_SyMax"/>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_EyMin"/>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="lineEdit_DxMax"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Xmin</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="lineEdit_FyMax"/>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="lineEdit_SxMax"/>
          </item>
          <item row="5" column="3">
           <widget class="QLineEdit" name="lineEdit_SyMin"/>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="lineEdit_NyMin"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Xmax</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Ymin</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="lineEdit_EyMax"/>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="lineEdit_DyMax"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_SxMin"/>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_FxMax"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_NxMin"/>
          </item>
          <item row="4" column="4">
           <widget class="QLineEdit" name="lineEdit_NyMax"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_DxMin"/>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="lineEdit_NxMax"/>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEdit_FyMin"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_FxMin"/>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEdit_DyMin"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_ExMax"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_ExMin"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Ymax</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>ENEA</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>FISE</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>DLR</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>NREL</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>SANDIA</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Comparison</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_compare">
           <property name="text">
            <string>Compare</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_iPtn0"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_iPtn1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>mean</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_mean"/>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>RMS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_RMS"/>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>PV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_PV"/>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_diffMin"/>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_diffMax"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="checkBox_mapDiff">
           <property name="text">
            <string>2D map &amp;&amp; profile plot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2DmapMin"/>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2DmapMax"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
